core/src/storage2/lazy/lazy_cell.rs

```diff
@@ -307,6 +308,33 @@ where
     pub fn get_mut(&mut self) -> Option<&mut T> {
         self.load_entry_mut().value_mut().into()
     }
+
+    /// Sets the value in this cell to `value`, without executing any reads.
+    ///
+    /// # Note
```

Comment:
Should note that this method should be preferred over get_mut in case the returned value is of no interest to the caller.